src/Standards/PEAR/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -41,16 +49,29 @@ public function register()
     public function process(File $phpcsFile, $stackPtr)
     {
         $tokens = $phpcsFile->getTokens();
-        $find   = Tokens::$methodPrefixes;
-        $find[] = T_WHITESPACE;
 
-        $commentEnd = $phpcsFile->findPrevious($find, ($stackPtr - 1), null, true);
+        $ignore   = Tokens::$emptyTokens;
+        $ignore[] = T_STATIC;
+
+        $scopeModifier = $phpcsFile->findPrevious($ignore, ($stackPtr - 1), null, true);
```

Comment:
Might be worth using `$phpcsFile->getMethodProperties($stackPtr)` to gather the method visibility + just check the `scope` index of the return value, instead of looking for it yourself. Makes a bit more future-proof.

Reply:
@gsherwood Done